cromulent/vocab.py

```diff
@@ -77,33 +77,43 @@ def register_instance(name, data):
-	"Watermarks": {"parent": LinguisticObject, "id": "300028749", "label": "Watermarks", "brief": True},
-
-	"_BriefText": {"parent": LinguisticObject, "id": "300418049", "label":"Brief Text"},
-	"MaterialStatement": {"parent": LinguisticObject, "id": "300010358", "label": "Material Statement", "brief": True},
-	"DimensionStatement": {"parent": LinguisticObject, "id": "300266036", "label": "Dimension Statement", "brief": True},
-	"CreditStatement": {"parent": LinguisticObject, "id": "300026687", "label": "Credit Statement", "brief": True},
-	"RightsStatement": {"parent": LinguisticObject, "id": "300055547", "label": "Rights Statement", "brief": True},
-	"EditionStatement": {"parent": LinguisticObject, "id":"300121294", "label": "Edition Statement", "brief": True},
-	"BiographyStatement": {"parent": LinguisticObject, "id":"300080102", "label": "Biography Statement", "brief": True},
-	"ProvenanceStatement": {"parent": LinguisticObject, "id":"300055863", "label": "Provenance Statement", "brief": True},
+	"Note": {"parent": LinguisticObject, "id":"300027200", "label": "Note", "brief": True},
 	"Description": {"parent": LinguisticObject, "id":"300411780", "label": "Description", "brief": True},
-	"PaginationStatement": {"parent": LinguisticObject, "id":"300200294", "label": "Pagination Statement", "brief": True},
-	"FoliationStatement": {"parent": LinguisticObject, "id":"300200662", "label": "Foliation Statement", "brief": True},
-	"CultureStatement": {"parent": LinguisticObject, "id": "300055768", "label": "Culture Statement", "brief": True},
-	"BibliographyStatement": {"parent": LinguisticObject, "id": "300026497", "label": "Bibliography Statement", "brief": True},
-	"AcquisitionStatement": {"parent": LinguisticObject, "id": "300157782", "label": "Acquisition Statement", "brief": True},
-	"LanguageStatement": {"parent": LinguisticObject, "id": "300386046", "label": "Language Statement", "brief": True},
-	"AppraisalStatement": {"parent": LinguisticObject, "id": "300027474", "label": "Appraisal Statement", "brief": True},
-	"SourceStatement": {"parent": LinguisticObject, "id": "300404764", "label": "Source Statement", "brief": True},
+	"_BriefText": {"parent": LinguisticObject, "id": "300418049", "label":"Brief Text"},
 	"Abstract": {"parent": LinguisticObject, "id":"300026032", "label": "Abstract", "brief": True},
-	"Note": {"parent": LinguisticObject, "id":"300027200", "label": "Note", "brief": True},
 	"Annotation": {"parent": LinguisticObject, "id":"300026100", "label":"Annotation"},
 	"Citation": {"parent": LinguisticObject, "id":"300311705", "label": "Citation", "brief": True},
-	"ExhibitionStatement": {"parent": LinguisticObject, "id":"300054766", "label":"Exhibition Statement", "brief": True},
+
+	"CreditStatement": {"parent": LinguisticObject, "id": "300026687", "label": "Credit Statement", "brief": True},
+	"RightsStatement": {"parent": LinguisticObject, "id": "300435434", "label": "Copyright/License Statement", "brief": True},
```

Comment:
While spot checking, I couldn't find this AAT term at http://vocab.getty.edu/aat/300435434. Is there delay between new terms being added and showing up in the web UI?

Reply:
Yes, they only load GraphDB every quarter (!) so it's coming but not there yet.

Reply:
(Also seems really unfortunate that not-found URIs like that are returning HTTP 200.